Dockerfile

```diff
@@ -1,13 +1,26 @@
-FROM norionomura/swiftlint:0.40.0 as builder
+ARG SWIFT_VERSION=5.3
 
-FROM swift:5.2.4-xenial-slim
+FROM norionomura/swiftlint:0.40.3_swift-$SWIFT_VERSION as builder
+
+FROM swift:$SWIFT_VERSION-xenial-slim as base
 
 COPY --from=builder /usr/bin/swiftlint /usr/bin/swiftlint
 COPY --from=builder /usr/lib/libsourcekitdInProc.so /usr/lib/libsourcekitdInProc.so
-
 COPY docs /docs
-COPY target/graalvm-native-image/codacy-swiftlint /workdir/
 RUN adduser --uid 2004 --disabled-password --gecos "" docker
+
+FROM base as dev
+
+RUN apt-get update && apt-get install -y openjdk-8-jre
```

Comment:
Codacy found an issue: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/gh/codacy/codacy-swiftlint/pullRequest?prid=6224691)